<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSotr</class>
 <widget class="QDialog" name="DialogSotr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Сотрудник</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabMain">
      <attribute name="title">
       <string>Общее</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayoutMain">
         <item row="0" column="0" rowspan="2">
          <widget class="QGroupBox" name="groupBoxFoto">
           <property name="title">
            <string>Фотография</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="spacing">
             <number>5</number>
            </property>
            <property name="margin">
             <number>5</number>
            </property>
            <item>
             <widget class="QToolButton" name="toolButtonFoto">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="qrc.qrc">
                <normaloff>:/icon/icons/nofoto.png</normaloff>:/icon/icons/nofoto.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>77</width>
                <height>99</height>
               </size>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1" rowspan="2" colspan="3">
          <widget class="QGroupBox" name="groupBoxAdress">
           <property name="title">
            <string>Место проживания</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_10">
            <item row="0" column="0">
             <widget class="QLabel" name="labelPost">
              <property name="text">
               <string>Индекс</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="spinBoxPost">
              <property name="maximum">
               <number>999999</number>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="labelHouse">
              <property name="text">
               <string>Дом</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QSpinBox" name="spinBoxHouse">
              <property name="maximum">
               <number>9999</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="labelCity">
              <property name="text">
               <string>Город</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEditCity"/>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="labelCase">
              <property name="text">
               <string>Квартира</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QSpinBox" name="spinBoxCase">
              <property name="maximum">
               <number>9999</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="labelStreet">
              <property name="text">
               <string>Улица</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="lineEditStreet"/>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="labelFlat">
              <property name="text">
               <string>Корпус</string>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QSpinBox" name="spinBoxFlat">
              <property name="maximum">
               <number>999</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QGroupBox" name="groupBoxSotr">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Личные данные</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="0" column="0">
             <widget class="QLabel" name="labelLastName">
              <property name="text">
               <string>Фамилия</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="labelFirstName">
              <property name="text">
               <string>Имя</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEditFirstName"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="labelMidName">
              <property name="text">
               <string>Отчество</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="lineEditMidName"/>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEditLastName"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="labelBirthday">
              <property name="text">
               <string>Дата рож.</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QDateEdit" name="dateEditBirthday">
              <property name="displayFormat">
               <string>dd.MM.yyyy</string>
              </property>
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="labelSex">
              <property name="text">
               <string>Пол</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QComboBox" name="comboBoxSex"/>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="2" colspan="2">
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Телефоны</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="TableSqlWidget" name="tableViewTel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="3" column="1">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabWork">
      <attribute name="title">
       <string>Место работы</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="labelTabNum">
           <property name="text">
            <string>Табельный</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="labelCategory">
           <property name="text">
            <string>Категория</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labelEdu">
           <property name="text">
            <string>Образование</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelOtdel">
           <property name="text">
            <string>Подразделение</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelDolg">
           <property name="text">
            <string>Должность</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="labelSoc">
           <property name="text">
            <string>Соц. положен.</string>
           </property>
          </widget>
	 </item>
	 <item row="0" column="1" colspan="2">
	 <widget class="QSpinBox" name="spinBoxTabNum">
	  <property name="maximum">
	   <number>99999999</number>
	  </property>
	 </widget>
	</item>
	<item row="1" column="1" colspan="2">
	<widget class="QCBoxButton" name="comboBoxDolgn"/>
       </item>
         <item row="2" column="1" colspan="2">
          <widget class="QCBoxButton" name="comboBoxOtdel"/>
         </item>

         <item row="1" column="4">
          <widget class="QComboBox" name="comboBoxSoc"/>
         </item>

         <item row="2" column="3">
          <widget class="QLabel" name="labelCond">
           <property name="text">
            <string>Условия труда</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="2">
          <widget class="QCBoxButton" name="comboBoxEdu"/>
         </item>
         <item row="0" column="4">
          <widget class="QComboBox" name="comboBoxCategory"/>
         </item>
         <item row="2" column="4">
          <widget class="QComboBox" name="comboBoxCond"/>
         </item>
         <item row="4" column="0" colspan="5">
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Предыдущее место работы</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="5">
          <widget class="TableSqlWidget" name="tableViewPrevWork"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSpec">
      <attribute name="title">
       <string>Специальный</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="4" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBoxPasport">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Паспортные данные</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="1" column="1" colspan="2">
           <widget class="QDateEdit" name="dateEditPaspDate">
            <property name="displayFormat">
             <string>dd.MM.yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="labelPasport">
            <property name="text">
             <string>Паспорт</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QLineEdit" name="lineEditPaspPlace"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelPaspPlace">
            <property name="text">
             <string>Кем выдан</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelPaspDate">
            <property name="text">
             <string>Когда выдан</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelINN">
            <property name="text">
             <string>ИНН</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spinBoxPaspSer">
            <property name="maximum">
             <number>9999</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QSpinBox" name="spinBoxPaspNum">
            <property name="maximum">
             <number>99999999</number>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="2">
           <widget class="QSpinBox" name="spinBoxINN">
            <property name="maximum">
             <number>999999999</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" rowspan="2" colspan="2">
        <widget class="QGroupBox" name="groupBoxOther">
         <property name="title">
          <string>Другое</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBoxVeteran">
            <property name="text">
             <string>Ветеран труда</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkBoxTrauma">
            <property name="text">
             <string>Травма на производстве</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="checkBoxProvZab">
            <property name="text">
             <string>Профзаболевание</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelInvalid">
            <property name="text">
             <string>Инвалидность</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QComboBox" name="comboBoxInvalid"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QGroupBox" name="groupBoxHouse">
         <property name="title">
          <string>Жилье</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBoxNeedHome">
            <property name="text">
             <string>Нуждается в жилье</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelNeedDate">
            <property name="text">
             <string>Жилье предоставленено</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QDateEdit" name="dateEditNeedDate">
            <property name="displayFormat">
             <string>dd.MM.yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="2" rowspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Даты рождения детей:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="TableSqlWidget" name="tableViewChild">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabProf">
      <attribute name="title">
       <string>Профсоюз</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="1">
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="2" column="0">
          <widget class="QLabel" name="label_31">
           <property name="text">
            <string>Номер профбилета</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="checkBoxProfbiletOut">
           <property name="text">
            <string>Выдан</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0" colspan="2">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="spinBoxNumProfbilet"/>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBoxMember"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labelMember">
           <property name="text">
            <string>Член профсоюза</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="labelStage">
           <property name="text">
            <string>Профсоюзный стаж</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDateEdit" name="dateEditDateIn">
           <property name="displayFormat">
            <string>dd.MM.yyyy</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEditOrhProfbilet"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelDateIn">
           <property name="text">
            <string>Дата вступления/выбытия</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labelOrgProfbilet">
           <property name="text">
            <string>Организация выдавшая профбилет</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="labelProfActiv">
           <property name="text">
            <string>Профсоюзный актив</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="labelElected">
           <property name="text">
            <string>Избран</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QComboBox" name="comboBoxElected"/>
         </item>
         <item row="7" column="1">
          <widget class="QComboBox" name="comboBoxProfActiv"/>
         </item>
         <item row="4" column="1">
          <widget class="QCheckBox" name="checkBoxAdmitted">
           <property name="text">
            <string>Принят впервые</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QCheckBox" name="checkBoxKadrSotr">
           <property name="text">
            <string>Кадровый работник</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QSpinBox" name="spinBoxStage"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableSqlWidget</class>
   <extends>QTableView</extends>
   <header>tablesqlwidget.h</header>
  </customwidget>
  <customwidget>
   <class>QCBoxButton</class>
   <extends>QComboBox</extends>
   <header>qcboxbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>toolButtonFoto</tabstop>
  <tabstop>lineEditLastName</tabstop>
  <tabstop>lineEditFirstName</tabstop>
  <tabstop>lineEditMidName</tabstop>
  <tabstop>dateEditBirthday</tabstop>
  <tabstop>comboBoxSex</tabstop>
  <tabstop>spinBoxPost</tabstop>
  <tabstop>lineEditCity</tabstop>
  <tabstop>lineEditStreet</tabstop>
  <tabstop>spinBoxHouse</tabstop>
  <tabstop>spinBoxCase</tabstop>
  <tabstop>spinBoxFlat</tabstop>
  <tabstop>tableViewTel</tabstop>
  <tabstop>spinBoxTabNum</tabstop>
  <tabstop>comboBoxDolgn</tabstop>
  <tabstop>comboBoxOtdel</tabstop>
  <tabstop>comboBoxEdu</tabstop>
  <tabstop>comboBoxCategory</tabstop>
  <tabstop>comboBoxSoc</tabstop>
  <tabstop>comboBoxCond</tabstop>
  <tabstop>tableViewPrevWork</tabstop>
  <tabstop>spinBoxPaspSer</tabstop>
  <tabstop>spinBoxPaspNum</tabstop>
  <tabstop>dateEditPaspDate</tabstop>
  <tabstop>lineEditPaspPlace</tabstop>
  <tabstop>spinBoxINN</tabstop>
  <tabstop>checkBoxNeedHome</tabstop>
  <tabstop>dateEditNeedDate</tabstop>
  <tabstop>checkBoxVeteran</tabstop>
  <tabstop>checkBoxTrauma</tabstop>
  <tabstop>checkBoxProvZab</tabstop>
  <tabstop>comboBoxInvalid</tabstop>
  <tabstop>tableViewChild</tabstop>
  <tabstop>comboBoxMember</tabstop>
  <tabstop>dateEditDateIn</tabstop>
  <tabstop>spinBoxNumProfbilet</tabstop>
  <tabstop>lineEditOrhProfbilet</tabstop>
  <tabstop>checkBoxProfbiletOut</tabstop>
  <tabstop>checkBoxAdmitted</tabstop>
  <tabstop>checkBoxKadrSotr</tabstop>
  <tabstop>spinBoxStage</tabstop>
  <tabstop>comboBoxProfActiv</tabstop>
  <tabstop>comboBoxElected</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="qrc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogSotr</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>283</x>
     <y>387</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
